boards/nrf52840dongle/doc.txt

```diff
@@ -21,7 +29,7 @@ reset button as well as 15 configurable external pins.
 - [nRF52840 web page](https://www.nordicsemi.com/?sc_itemid=%7BCDCCA013-FE4C-4655-B20C-1557AB6568C9%7D)
 - [documentation and hardware description](https://infocenter.nordicsemi.com/topic/ug_nrf52840_dongle/UG/nrf52840_Dongle/intro.html?cp=3_0_5)
 
-### Flash the board
+### Flash the board {#nrf52840dongle_flash}
```

Comment:
I think you can't reference sub-section on non-pages. :-/

Reply:
@jeandudey found a solution for this in e.g. [`cc1352p-launchpad`](https://github.com/RIOT-OS/RIOT/blob/master/boards/cc1352p-launchpad/doc.txt#L13)

Reply:
actually based on:

https://github.com/RIOT-OS/RIOT/blob/930e760143b4a85762aae533331bdda3218e82f4/cpu/esp32/doc.txt#L138

:eyes: